cooltools/lib/common.py

```diff
@@ -532,15 +532,14 @@ def wrapper(*args, **kwargs):
                 mymap = pool.imap
             else:
                 mymap = map
-
-            func(*args, **kwargs, map=mymap)
-
+            try:
+                result = func(*args, **kwargs, map=mymap)
```

Comment:
so, `pool_decorator` can decorate function that both have `nproc` and `map` as key word arguments, correct ?

Reply:
Yes, this should be documented.